packages/web/src/settings/_links.scss

```diff
@@ -0,0 +1,36 @@
+@use '@tokens' as tokens;
+
+$default-visited: tokens.$action-link-primary-visited;
+
+$variants: (
+    primary: (
+        default: tokens.$action-link-primary-default,
+        hover: tokens.$action-link-primary-hover,
+        active: tokens.$action-link-primary-active,
+        disabled: tokens.$action-link-primary-disabled,
+    ),
+    secondary: (
+        default: tokens.$action-link-secondary-default,
+        hover: tokens.$action-link-secondary-hover,
+        active: tokens.$action-link-secondary-active,
+        disabled: tokens.$action-link-secondary-disabled,
+    ),
+    inverted: (
+        default: tokens.$action-link-inverted-default,
+        hover: tokens.$action-link-inverted-hover,
+        active: tokens.$action-link-inverted-active,
+        disabled: tokens.$action-link-inverted-disabled,
+    ),
+);
+
+$visited-variants: (
+    primary: (
+        visited: tokens.$action-link-primary-visited,
```

Comment:
I'd like to see this map merged into `$variants` and decide (parametrically) what states to return in the `variants()` mixin. Does it make things too complicated? What do you think? 🤔

Reply:
Sure we can do this. Do you want it to be possible to choose each variant value item? Or just boolean for visited?

Reply:
Another way could be to keep two naps, but one (headings) would be an extension of the other (default links).